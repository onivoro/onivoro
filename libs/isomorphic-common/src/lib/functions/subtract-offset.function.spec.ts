import { subtractOffset } from './subtract-offset.function';

describe('subtractOffset', () => {
    describe('GIVEN input is partial or full ISO8601', () => {
        it.each([
            '2023-12-23',
            '2023-12-23T00:00:00.000',
        ])('%j', (input) => {
            expect(subtractOffset(input)).toMatchSnapshot();
        });
    });
});