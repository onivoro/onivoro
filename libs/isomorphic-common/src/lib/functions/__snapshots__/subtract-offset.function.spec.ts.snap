// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`subtractOffset GIVEN input is partial or full ISO8601 "2023-12-23" 1`] = `2023-12-22T18:00:00.000Z`;

exports[`subtractOffset GIVEN input is partial or full ISO8601 "2023-12-23T00:00:00.000" 1`] = `2023-12-23T00:00:00.000Z`;
